cmd/up/activate.go

```diff
@@ -115,6 +115,11 @@ func (up *upContext) activate() error {
 		lastPodUID = up.Pod.UID
 	}
 
+	if err := up.waitUntilAppIsAwaken(ctx, app); err != nil {
```

Comment:
That code doesn't start any actual synchronization as the dev container is not still created. The idea is to "block" waiting for it until the last moment before the dev mode starts, so hopefully, it is not needed to wait too much because the resource is already awaken when we hit this code